FraudProtection/build.gradle

```diff
@@ -40,7 +40,7 @@ dependencies {
         implementation files('libs/android-magnessdk-5.3.0.jar')
     }
 
-
+    api project(':CorePayments')
```

Comment:
Not sure if this works the same on the Android side but should we call out this dependency in the changelog?

Reply:
Yeah we don't usually mention dependency additions on Android since they're declared in the Gradle file and become transitive dependencies in a merchant project. How do y'all do it on iOS?